src/__tests__/cli.ts

```diff
@@ -0,0 +1,490 @@
+      stdout: /1 unused dependencies.*@test\/dependency/s,
+    },
+    {
+      name: 'should not report issues when everything is used',
+      files: [
+        {
+          name: 'package.json',
+          content:
+            '{ "main": "index.js", "dependencies": { "@test/dependency": "1.0.0" } }',
+        },
+        {
+          name: 'index.js',
+          content: `
+import foo from './foo';
+import bar from './bar';
+`,
+        },
+        { name: 'foo.js', content: '' },
+        { name: 'bar.js', content: 'import test from "@test/dependency"' },
+      ],
+      exitCode: 0,
+      stdout: /There don't seem to be any unimported files./,
+    },
+    {
+      name: 'should use all variants of import/export',
+      files: [
+        {
+          name: 'package.json',
+          content: '{ "main": "index.js" }',
+        },
+        {
+          name: 'index.js',
+          content: `import a from './a'`,
+        },
+        {
+          name: 'a.js',
+          content: `
+import {b as a} from './b'
+const promise = import('./d')
+export {a}
+export {b} from './b'
+export * from './c'
+export default promise
+`,
+        },
+        { name: 'b.js', content: 'export const b = 2;' },
+        { name: 'c.js', content: 'const c = 3; export {c}' },
+        { name: 'd.js', content: 'export default 42' },
+      ],
+      exitCode: 0,
+      stdout: /There don't seem to be any unimported files./,
+    },
+    {
+      name: 'should identify ts paths imports',
+      files: [
+        { name: 'package.json', content: '{ "main": "index.ts" }' },
+        { name: 'index.ts', content: `import foo from '@root/foo';` },
+        { name: 'foo.ts', content: '' },
+        { name: 'bar.ts', content: '' },
+        {
+          name: 'tsconfig.json',
+          content: '{ "compilerOptions": { "paths": { "@root": ["."] } } }',
+        },
+      ],
+      exitCode: 1,
+      stdout: /1 unimported files.*bar.ts/s,
+    },
+    {
+      name: 'should identify monorepo-type sibling modules',
+      baseDir: 'packages/A',
+      files: [
+        {
+          name: 'packages/A/package.json',
+          content:
+            '{ "main": "index.js", "repository": { "directory": "path/goes/here" } }',
+        },
+        {
+          name: 'packages/A/index.js',
+          content: `import foo from 'B/foo';`,
+        },
+        { name: 'packages/B/foo.js', content: '' },
+        { name: 'packages/C/bar.js', content: '' },
+      ],
+      exitCode: 0,
+      stdout: /There don't seem to be any unimported files./,
+    },
+    {
+      name: 'should support root slash import in meteor project',
+      files: [
+        {
+          name: 'package.json',
+          content:
+            '{ "meteor" : { "mainModule": { "client": "client.js", "server": "server.js" } } }',
+        },
+        { name: 'client.js', content: `import foo from '/foo';` },
+        { name: 'server.js', content: '' },
+        { name: '.meteor', content: '' },
```

Comment:
```suggestion
        { name: '.meteor/release', content: 'METEOR@1.11.1' },
```

Reply:
Does this need content like the other suggestion?

Reply:
Nah, you're too fast :innocent: . I removed the contents from the other one as well.